ccd-definition/CaseEventToComplexTypes/CreateClaim-nonprod.json

```diff
@@ -170,5 +170,69 @@
     "ListElementCode": "OrgPolicyReference",
     "FieldDisplayOrder": 4,
     "DisplayContext": "OPTIONAL"
+  },
+  {
+    "ID": "Respondent2ServiceAddress",
+    "CaseEventID": "CREATE_CLAIM",
+    "CaseFieldID": "respondentSolicitor2ServiceAddress",
+    "ListElementCode": "hasServiceAddress",
```

Comment:
I would maybe just call this field required. Then it's respondentSolicitor2ServiceAddress.required

Reply:
actioned